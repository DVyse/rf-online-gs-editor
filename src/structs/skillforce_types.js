export const SPELL = 'spell';
export const SKILL = 'skill';
export const CLASS_SKILL = 'class_skill';
export const BULLET = 'bullet';
export const POTION = 'potion';
