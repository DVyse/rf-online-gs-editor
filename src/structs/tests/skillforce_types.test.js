import * as skillforceTypes from '../skillforce_types';

describe('Structs', () => {
  describe('SkillforceTypes', () => {
    it('should have skillforceTypes', () => {
      expect(skillforceTypes).toMatchSnapshot();
    });
  });
});
