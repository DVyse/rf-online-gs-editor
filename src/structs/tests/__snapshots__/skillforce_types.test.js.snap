// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Structs SkillforceTypes should have skillforceTypes 1`] = `
Object {
  "BULLET": "bullet",
  "CLASS_SKILL": "class_skill",
  "POTION": "potion",
  "SKILL": "skill",
  "SPELL": "spell",
}
`;
